PennyMe/Artwork.swift

```diff
@@ -54,9 +56,24 @@ class Artwork: NSObject, MKAnnotation {
         title = properties["name"] as? String
         locationName = (properties["address"] as? String)!
         link = (properties["external_url"] as? String)!
-        status = (properties["status"] as? String)!
         last_updated = (properties["last_updated"] as? String)!
         id = String((properties["id"] as? Int)!)
+        // machine is per default active and unvisited
+        machineStatus = "active"
+        if let statusTemp = properties["active"] as? Bool {
+            if !statusTemp {
+                machineStatus = "retired"
+            }
+        }
+        if let statusTemp = properties["active"] as? String {
+            machineStatus = statusTemp
+        }
+        // the status is not read from the dictionary, but is set to unvisited unless the machine is not active
+        status = "unvisited"
+        if machineStatus != "active" {
```

Comment:
This induces a dependency between both statuses which we should definitely avoid imho.

status here refers to the user-collection status which cannot be retired. It should always default to "unvisited" but then can be overwritten with variables in the personal collection dict